### ISSUE TITLE
I am going to buy led matrix device.

### ISSUE THREAD

**Issue by andrei0807** [hzeller/rpi-rgb-led-matrix#843]
I live in china.
I am going to led matrix device in china.
https://www.adafruit.com/product/3211
But I can not find this product in taobao or Alibaba.
Can you send me product's link.
Thanks.

**Comment by hzeller** [comment]
If you look for HUB75 matrixes, you probably find something.

**Comment by andrei0807** [comment]
@hzeller
I did not find device.
 Can you send me link?

**Comment by hzeller** [comment]
Search for "hub75" on aliexpress for instance. You'll find tons of LED matrices.

**Comment by postelontwerpen** [comment]
here you go:
P2.5 Indoor SMD2121 Full Color Led Display Module 1/32 Scan 320x160mm
[https://www.aliexpress.com/item/P2-5-Indoor-SMD2121-Full-Color-Led-Display-Module-1-32-Scan-320x160mm-Super-High-Definition/32845686589.html?spm=a2g0s.9042311.0.0.3da24c4dC5F3eD
](url)

Power Supply Transformers 5V 400W
[https://www.aliexpress.com/item/New-5V-2A-3A-4A-5A-6A-8A-10A-12A-20A-30A-40A-60A-SwitchLED-Power/32641542774.html?spm=a2g0s.9042311.0.0.3da24c4dC5F3eD
](url)

Flat Ribbon Data Cable For LED Display 16P 20cm
[https://www.aliexpress.com/item/5Pcs-FC-10P-14P-16P-20P-26P-34P-40P-Gray-Flat-Ribbon-Data-Cable-For-LED/32849047343.html?spm=a2g0s.9042311.0.0.3da24c4dC5F3eD](url)

RGB Matrix Panel Drive Board For Raspberry Pi
[https://www.electrodragon.com/product/rgb-matrix-panel-drive-board-raspberry-pi/](url)

25V 3300UF 25 V / 3300 UF Aluminum Electrolytic Capacitors
[https://www.aliexpress.com/item/20PCS-25V-3300UF-25-V-3300-UF-Aluminum-Electrolytic-Capacitors-Size-16-25-3300UF-25V-Electrolytic/32877266431.html?spm=2114.search0204.3.34.59f67586Td48bN&ws_ab_test=searchweb0_0,searchweb201602_3_10065_10068_318_319_5727315_10696_10084_10083_10618_452_10304_10307_10820_532_10821_10301_5727215_204_10843_10059_10884_10887_100031_320_10103,searchweb201603_1,ppcSwitch_0_ppcChannel&algo_expid=fea176a5-28a5-4cbc-806a-5759c7aa1d81-5&algo_pvid=fea176a5-28a5-4cbc-806a-5759c7aa1d81&priceBeautifyAB=0](url)

**Comment by andrei0807** [comment]
@partofthething 
Thanks for your response.
https://www.adafruit.com/product/3211
I want this device in aliexpress.

**Comment by postelontwerpen** [comment]
Yeah, be carefull with this. Adafruit uses an own ledpanel (other pin-addressing) that could be different from those you referred to at Aliexpress. There are others, here in this Issue-list, that tried and failed. I did not build de Adafruit version myself. At least, I tested succesfully with the Adafruit Hat and two panels I described (url) in my previous reaction.

Just to be more save with Aliexpress-panels : build it with the description you can find in this Repo, instead of Adafruits. And use a RPi3 B+ or lower for now. Or buy the Adafruit stuff and build that one.

**Comment by jaygarcia** [comment]
> @partofthething
> Thanks for your response.
> https://www.adafruit.com/product/3211
> I want this device in aliexpress.

@andrei0807 if  you're using a general raspberry pi, i suggest the electrodragon daughter board.

https://www.electrodragon.com/product/rgb-matrix-panel-drive-board-raspberry-pi/

**Comment by marcmerlin** [comment]
With apologies to everyone, this project is doing a one time cleanup of 
all the bugs that have not been addressed and closed in the last 10 years.

If you filed an issue to ask for help, please do not re-open this one or a new
bug, but go here instead: https://rpi-rgb-led-matrix.discourse.group/

Please be mindful that this project does not have any full time maintainers
or anyone paid. You should not expect personal help for your own projects or
answers to programming questions but you can ask for help with panels that
aren't working, setup questions, or anything that may be a bug here:
https://rpi-rgb-led-matrix.discourse.group/
If you need programming help and are willing to pay someone for it, please
state so in your message, and someone may be interested.

If you did report an actual bug that is still not fixed today, please accept
our apologies for this being mass closed, you can absolutely send a followup
and re-open the issue, or open a new up to date one as applicable.
Note that we very much welcome help, so if you can help fix your bug, you will
be a superstar :)

If your bug was linked to a pull request that was not resolved, please update
the pull request and the issue, and we will do our best to review them. We do
thank you for your undertanding that we have very limited free time, so any
extra work you do along with your bug report or PR request will be very much
appreciated and improve the odds that they will be looked at.

If you want to help contribute as a python or C# or any other language bindings,
you are also a superstar, please state so in the issue and we'll do our best to
give you the access and reviews you need.

For anything related to PWM panel support, please make sure you read README and
then refer to https://github.com/hzeller/rpi-rgb-led-matrix/issues/466 . 
Please do not comment if you're haven't read everything and/or if you're just
asking when your panel is going to be supported. Those bugs are if you're 
able to contribute to the effort.

For rPI5: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1603

For anything python, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
For anything C#, please link your bug to: https://github.com/hzeller/rpi-rgb-led-matrix/issues/1828
